service/pom.xml

```diff
@@ -145,6 +145,23 @@
 			<artifactId>h2</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
Das Grundproblem beim Pull-Request ist,
dass jede Codestelle unterscheiden muss, ob der alter Code verwendet wird oder JOOQ und dass sollte nicht sein.
Die Datenbankdetails sollten in dezidierten Klassen ausgelagert sein und stattdessen sollten generele SQL-*-Services passende Implementierungen von SQL-spezifischen Code über eine allgemeine Schnittstelle bereitstellen.